src/openforms/submissions/templates/admin/submissions/submission/change_form.html

```diff
@@ -1,6 +1,13 @@
 {% extends 'admin/change_form.html' %}
 {% load i18n solo_tags %}
 
+{% block object-tools-items %}
+{{ block.super }}
+  <li>
+    <a href="{% url 'admin:logs-evaluated-logic' submission_id=original.pk %}">{% trans "Logging" %}</a>
```

Comment:
```suggestion
    <a href="{% url 'admin:logs-evaluated-logic' submission_id=original.pk %}">{% trans "Logic logs" %}</a>
```
the rest of the logs are still on this same page